account_financial_test_data/__openerp__.py

```diff
@@ -0,0 +1,42 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    OpenERP, Open Source Management Solution
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program. If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+
+{
+    'name': 'Account financial test data',
+    'version': '9.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': "Taktik, Odoo Community Association (OCA)",
+    'category': 'Other',
+    'depends': [
+        "base",
+        "account",
+        "account_accountant",
+        "l10n_fr",
+    ],
+    'demo': [],
+    'data': [
```

Comment:
It depends.
If i put the xml data in "demo", it means all others demo data will be added.
Maybe for clarity, it is more flexible to let it as it is.
It will let all the users the choice.
But if it make no sense, I m ready to change it.